experimental/stats/metricregistry_test.go

```diff
@@ -263,6 +275,8 @@ func newFakeMetricsRecorder(t *testing.T) *fakeMetricsRecorder {
 		case MetricTypeIntCount:
 		case MetricTypeIntHisto:
 		case MetricTypeIntGauge:
+		case MetricTypeIntUpDownCount:
+		case MetricTypeIntAsyncGauge:
```

Comment:
Unlike Java, switch cases in Go don't fall through. This change is changing the behaviour. However, I don't see any tests failing, so maybe it's safe to remove this for loop completely.